### ISSUE TITLE
Dark and Light mode

### ISSUE THREAD

**Issue by rawatji001** [Omkar-Sonawane-23/Tourism#200]
Your current website lacks a light and dark mode feature, which is essential for improving user experience and accessibility. This issue aims to implement a toggleable light/dark mode functionality to allow users to choose their preferred theme.

**Comment by anniepauline** [comment]
I'd like to work on this issue

**Comment by sameer-soni** [comment]
@anniepauline okay, assigning you to this issue then.

**Comment by anniepauline** [comment]
working on it
